msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -180,7 +177,7 @@ POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/messages
 HTTP/1.1 200 OK
 
 {
-   "@odata.context":"https://graph.microsoft.com/beta/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
+   "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
```

Comment:
If you are using placeholder values, then use the format 'team-id' and 'channel-id'. If you are putting down illustrative values, then put a real team id and channel id in here and elsewhere.